pyproject.toml

```diff
@@ -23,15 +23,18 @@ dependencies = [
   "tbparse~=0.0.9",
   "toml~=0.10.2",
   "kubernetes~=34.1",
-  "pydantic~=2.12",
```

Comment:
Why this downgrade is needed? We should try using latest possible versions to avoid potential security issues.